source/beamline/CLS/CLSValves.h

```diff
@@ -16,6 +23,9 @@ class CLSValves : public CLSTriStateGroup
 	/// Destructor.
 	virtual ~CLSValves();
 
+	/// helper function to create a beam on action list
+	AMListAction3* createBeamOnActionList();
+
```

Comment:
That might be a safety thing. I'm not sure we'd want to turn off beam by attempting to close all the valves.